tests/tests_lvm_pool_shared.yml

```
@@ -0,0 +1,115 @@
+---
+- name: Test LVM shared pools
+  hosts: all
+  become: true
+  vars:
+    storage_safe_mode: false
+    storage_use_partitions: true
+    mount_location1: '/opt/test1'
+    volume1_size: '4g'
+
+  tasks:
+    - name: Create cluster
+      ansible.builtin.include_role:
+        name: fedora.linux-system-roles.ha_cluster
+      vars:
+        ha_cluster_cluster_name: rhel9-1node
+        # Users should vault-encrypt the password
+        ha_cluster_hacluster_password: hapasswd
+        ha_cluster_extra_packages:
+          - dlm
+          - lvm2-lockd
+        ha_cluster_cluster_properties:
+          - attrs:
+              # Don't do this in production
+              - name: stonith-enabled
+                value: 'false'
+        ha_cluster_resource_primitives:
+          - id: dlm
+            agent: 'ocf:pacemaker:controld'
+            instance_attrs:
+              - attrs:
+                  # Don't do this in production
+                  - name: allow_stonith_disabled
+                    value: 'true'
+          - id: lvmlockd
+            agent: 'ocf:heartbeat:lvmlockd'
+        ha_cluster_resource_groups:
+          - id: locking
+            resource_ids:
+              - dlm
+              - lvmlockd
+
+    - name: Run the role
+      include_role:
+        name: fedora.linux-system-roles.storage
```

```suggestion
        name: linux-system-roles.storage
```
have to use the old name with the tests for this role

---

Fixed.